### DISCUSSION TITLE
Attributes assignment on mobile

### DISCUSSION THREAD

**Discussion by Gleaped** [TriliumNext/Trilium#4862] [Q&A]
I just started running trillium next via docker on unraid and I cannot for the life of me figure out a way to assign attributes on the mobile view. What am I missing here? this seems like a pretty basic feature so it must be there?

Additionally how best can I format images in a note?

**Comment by meichthys** [comment]
Currently you must use the full desktop or web view to work with attributes. This will hopefully improve soon.
